pandas/_libs/lib.pyx

```diff
@@ -286,7 +287,9 @@ def fast_unique_multiple(list arrays, sort: bool = True):
         try:
             uniques.sort()
         except TypeError:
-            # TODO: RuntimeWarning?
+            warnings.warn(
+                "Not sure what will be a helpful error message", RuntimeWarning
```

Comment:
I would love if someone could give me an idea, on what will be a helpful warning message here.

Reply:
i know i suggested searching for TODO comments; i should have specified that the idea would to aim for ones that you _can_ solve, ideally that push your limits.  From a reviewer's perspective, this is pretty similar to the existing TODO comment

Reply:
> From a reviewer's perspective, this is pretty similar to the existing TODO comment

Haven't thought of that, I can see it now.